include/mixer.h

```diff
@@ -230,7 +240,7 @@ class MixerChannel {
 	// >0. Still work in progress and thus disabled for now.
 	int offset[2] = {0, 0};
 	int sample_rate = 0u;
-	AudioFrame volmul = {1.0f, 1.0f};
+	AudioFrame volume_gain = {1.0f, 1.0f};
```

Comment:
Not 100% sure what you mean, but I've done some further renamings.

Now we have these in the MixerChannel:

- `volume` (the raw value set by the caller)
- `volume_scale` (a secondary gain used to raise the OPL amplitude levels a little)
- `volume_gain` (calculated final gain (multiplier) that gets applied

Then in the mixer we have `master_volume` (self-explanatory).

Open to suggestions how to better name these. It can get a little confusing and I'm running out of ideas...

Reply:
Thanks, this answers it - 

I was looking at the new function names:

```c++
void SetVolumeScale(const float f);
void SetVolumeScale(const float left, const float right);
```

But then saw the member variable:

```c++
volume_gain ...
```

And was thinking we should use the word "scale" or "gain" in both places just to be consistent - but I see you've already got it w/ the `volume_scale` variable :+1: !

(resolving)